recipes/libgeotiff/all/conanfile.py

```diff
@@ -0,0 +1,74 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class LibgeotiffConan(ConanFile):
+    name = "libgeotiff"
+    description = "Libgeotiff is an open source library normally hosted on top " \
+                  "of libtiff for reading, and writing GeoTIFF information tags."
+    license = ["MIT", "BSD-3-Clause"]
+    topics = ("conan", "libgeotiff", "geotiff", "tiff")
+    homepage = "https://github.com/OSGeo/libgeotiff"
+    url = "https://github.com/conan-io/conan-center-index"
+    exports_sources = ["CMakeLists.txt", "patches/**"]
+    generators = "cmake"
+    settings = "os", "arch", "compiler", "build_type"
+    options = {"shared": [True, False], "fPIC": [True, False]}
+    default_options = {"shared": False, "fPIC": True}
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    @property
+    def _build_subfolder(self):
+        return "build_subfolder"
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def configure(self):
+        del self.settings.compiler.cppstd
+        if not self.options.shared or self.options["proj"].shared: # because PROJ is a C++ library with C API
+            del self.settings.compiler.libcxx
```

Comment:
```suggestion
            del self.settings.compiler.libcxx
            del self.settings.compiler.cppstd
```

Reply:
libgeotiff is a C library, there is no reason to keep cppstd I think.

Reply:
keep? my comment is about deleting cppstd

Reply:
It's already deleted above without conditions ;)

Reply:
:man_facepalming: